tests/system_test.py

```diff
@@ -282,17 +282,22 @@ def __init__(self, args, name=None, expect=EXIT_OK, **kwargs):
         @param expect: Raise error if process status not as expected at end of test:
             L{RUNNING} - expect still running.
             L{EXIT_OK} - expect process to have terminated with 0 exit status.
-            L{EXIT_FAIL} - expect process to have terminated with exit status 1.
+            L{EXIT_FAIL} - expect process to have terminated with exit status >= 1.
```

Comment:
and also `<0`, to be extra sure? Remember the exit codes of `-11` and so on in the report... Sometimes the exit code is `uint8` and sometimes it is `int8`. I never found what's the logic behind it.

Reply:
According to https://tldp.org/LDP/abs/html/exitcodes.html, codes for death after signal are `128 + signumber`, which, when represented as signed integer, produces those negative numbers.

Reply:
I'd leave this the way it was, that is, EXIT_FAIL means exit code == 1. If people want to check different codes, let them be explicit.

Reply:
This teardown() function handles expect codes like `SIGTERM (<0)` as seen here - https://github.com/skupperproject/skupper-router/blob/main/tests/system_tests_edge_router.py#L2700